src/components/Contentful/DynamicRichText.vue

```diff
@@ -0,0 +1,41 @@
+<script>
+import KvContentfulImg from '~/@kiva/kv-components/vue/KvContentfulImg';
+
+/**
+* This component takes an html string and renders it with vue
+* */
+
+export default {
+	props: {
+		html: {
+			type: String,
+			default: ''
+		}
+	},
+	computed: {
+		dynamicComponent() {
+			return {
+				template: `<div>${this.html}</div>`,
```

Comment:
this.html can include an image component: "kv-contentful-img"  In order to actually render that with vue we need the full compiler build.